DEVELOPMENT.md

```diff
@@ -50,3 +50,37 @@ The deploy script will push all of your packages, except for the `deployment` pa
 
 Run this command to deploy your code to AWS:
 `serverless deploy --aws-profile <PROFILE> --stage <STAGE>`
+
```

Comment:
Please disregard the diff for this file. This change should already be in `smart-develop`.

[Commit link for reference](https://github.com/awslabs/fhir-works-on-aws-deployment/commit/0d45d0dde0f5ae6ab3a7a176adeb6d3b37420e17)

[smart-develop](https://github.com/awslabs/fhir-works-on-aws-deployment/commits/smart-develop)